specifyweb/specify/models.py

```diff
@@ -7617,10 +7617,13 @@ class Meta:
         ordering = ()
         # unique_together = (('childcog'), ('childco'))
```

Comment:
are we going to keep the childco be unique? Just curious about how you got around the issues (I think it should be parentcog-childco be unique: https://github.com/specify/specify7/pull/5032#issuecomment-2236652508)

Reply:
Actually, this is fine.